managed-route53/build.f/particles-initial/particles/cftemplates/route53HostedZone.json.hbs

```diff
@@ -0,0 +1,16 @@
+{{#layout templateDescription="This template creates Route53 Hosted Zone"}}
+
+{{parameter "requiredString" logicalId="RouteHostName"}}
+{{parameter "optionalString" logicalId="RouteHostComment"}}
+
+{{#resource "route53HostedZone" logicalId="Route53HostedZone" }}
+ "Properties": {
```

Comment:
Properties should be part of resource definition. Can you move it to resource folder?